src/styles/blocks/stars.css

```
@@ -0,0 +1,34 @@
+.stars {
+  margin: 0;
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin: 0 2px;
+
+  background-image: url(../../images/star.svg);
+  background-repeat: no-repeat;
+  background-position: left center;
+}
+
+.stars--1 :first-child {
+  background-image: url(../../images/star-active.svg);
+}
+
+.stars--2 :nth-child(-n + 2) {
+  background-image: url(../../images/star-active.svg);
+}
+
+.stars--3 :nth-child(-n + 3) {
+  background-image: url(../../images/star-active.svg);
+}
+
+.stars--4 :nth-child(-n + 4) {
+  background-image: url(../../images/star-active.svg);
+}
+
+.stars--5 :nth-child(n) {
+  background-image: url(../../images/star-active.svg);
+}
```

```suggestion
.stars--1 :first-child,
.stars--2 :nth-child(-n + 2),
.stars--3 :nth-child(-n + 3),
.stars--4 :nth-child(-n + 4),
.stars--5 :nth-child(n) {
  background-image: url(../../images/star-active.svg);
}
```
You can try unit all selectors)